skills/README.md

```diff
@@ -0,0 +1,73 @@
+# Skills
+
+## Skills Object
+
+```
+{
+    "name": <name of skill>,
+    "awarded_on": <when skill was added>,
+    "awarded_by": <who added it>,
+    "awarded_for": <reason for adding it> 
+}
+```
+
+## **Requests**
+
+|                     Route                      |         Description          |
+| :--------------------------------------------: | :--------------------------: |
+|           [GET /skills](#get-skills)           | Returns skills in collection |
+| [POST /skills/:username](#post-skillsusername) |  Adds skills to collection   |

```

Comment:
How do we GET the skills of a particular member by their username?

Reply:
we can fetch the `skills` of a user and query DB with `/skills/skillId` and return an array of their skills ...